webui/Makefile

```diff
@@ -1,22 +1,27 @@
-.PHONY: default get-deps build clean test fmt check
-
-default: get-deps build
-
-clean:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-fmt:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-check:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
+.PHONY: all
+all:
+	make get-deps
```

Comment:
wat's the difference between make and $(MAKE)? it tells make that this is a submake? no upper makefile is calling this target thinking

Reply:
This should use `$(MAKE)`, which will allow this call to use the parallelism specified by the overall `make` invocation. So if you do `make -j4 all`, the `-j4` will be forwarded down into these.